sdk/formrecognizer/azure-ai-formrecognizer/tests/testcase.py

```diff
@@ -408,6 +415,13 @@ def assertFormPagesHasValues(self, pages):
                         self.assertBoundingBoxHasPoints(cell.bounding_box)
                         self.assertFieldElementsHasValues(cell.field_elements, page.page_number)
 
+            if page.selection_marks:
+                for selection_mark in page.selection_marks:
```

Comment:
If we leave `Text` empty, should we check here that no one assigned a value to it?